.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,46 @@
+name: Build and Test
+
+on: push
+
+jobs:
+  build-and-test:
+    name: Build and Test Web App
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
```

Comment:
You don't seem to need a matrix.
Move the steps to the steps list.

Reply:
Done in 6ccdf52